docs/networkpolicy-metrics.md

```diff
@@ -1,5 +1,6 @@
 # Network Policy Metrics
 
+<<<<<<< HEAD
```

Comment:
Looks like the rebase / merge has still some conflicts here

Reply:
I've fixed this conflict as well.
Thank you for the review.